api/oc_core_res.c

```diff
@@ -669,6 +672,9 @@ oc_core_is_SVR(const oc_resource_t *resource, size_t device)
   if (resource == NULL) {
     return false;
   }
+  if (device >= OC_ATOMIC_LOAD32(g_device_count)) {
+    return NULL;
```

Comment:
`NULL` -> `false`

Add call with invalid device index to the beginning of `CoreGetResourceIsSVR_P`

Reply:
Fixed